### PR TITLE
Make everything compatible with Fork 5

### PR DESCRIPTION
**Pull request by StijnVrolijk**
And fix some stupid stuff

### CODE REVIEW

File: src/Backend/Modules/Quotes/info.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
   </description>
   <authors>
     <author>
-      <name><![CDATA[Dylan Moerman]]></name>
```
Comment (carakas) [inline src/Backend/Modules/Quotes/info.xml:15]: Erased from existence

**Review by carakas — APPROVED** [review]

File: src/Backend/Modules/Quotes/info.xml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module>
   <name>Quotes</name>
-  <version>1.0.0</version>
+  <version>2.0.0</version>
   <requirements>
-    <minimum_version>4.0.0</minimum_version>
+    <minimum_version>5.0.6</minimum_version>
```
Comment (carakas) [inline src/Backend/Modules/Quotes/info.xml:6]: technically this is just 5.0.0

File: src/Backend/Modules/Quotes/info.xml (hunk 1 above)
Comment (StijnVrolijk) [reply src/Backend/Modules/Quotes/info.xml:15]: who you talking about

File: src/Backend/Modules/Quotes/Installer/Installer.php [hunk 3]
```diff
@@ -3,29 +3,30 @@
...
 
 final class Installer extends ModuleInstaller
 {
     public function install()
     {
         $this->addModule('Quotes');
 
-        $this->importSQL(dirname(__FILE__) . '/data/install.sql');
-        $this->importLocale(dirname(__FILE__) . '/data/locale.xml');
+        $this->importSQL(dirname(__FILE__) . '/Data/install.sql');
```
Comment (carakas) [inline src/Backend/Modules/Quotes/Installer/Installer.php:14]: the folder here is data so you need to change the casing or change the directory

**Review by carakas — CHANGES_REQUESTED** [review]

File: src/Backend/Modules/Quotes/info.xml (hunk 2 above)
Comment (StijnVrolijk) [reply src/Backend/Modules/Quotes/info.xml:6]: technically, the version i'm creating this on is 5.0.6. Who knows what sort of crazy bugfixes I'm relying on?

File: src/Backend/Modules/Quotes/Installer/Installer.php (hunk 3 above)
Comment (StijnVrolijk) [reply src/Backend/Modules/Quotes/Installer/Installer.php:14]: Fixed it in an amend

**Pull request merged**